### PR TITLE
[improvement](parquet-reader) Add some profile for parquet reader

### PR DESCRIPTION
**Pull request by morningman**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

1. Copy some profile from VArrowReader to FileArrowReader.
2. Refine the explain message of FileScanNode

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
4. Has unit tests been added: (Yes/No/No Need)
5. Has document been added or modified: (Yes/No/No Need)
6. Does it need to update dependencies: (Yes/No)
7. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/vec/exec/file_text_scanner.h [hunk 1]
```diff
@@ -42,6 +42,9 @@ class FileTextScanner final : public FileScanner {
     Status get_next(Block* block, bool* eof) override;
     void close() override;
 
+protected:
+    void _init_profiles(RuntimeProfile* profile) override {}
```
Comment (wsjz) [inline be/src/vec/exec/file_text_scanner.h:46]: the override is useless?

**Review by wsjz — COMMENTED** [review]
LGTM

File: be/src/vec/exec/file_text_scanner.h (hunk 1 above)
Comment (morningman) [reply be/src/vec/exec/file_text_scanner.h:46]: I just want to make it more explicitly.

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**